### PR TITLE
Add recipe for inpoly

### PR DESCRIPTION
**Pull request by xylar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/inpoly/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    INPOLY returns the "inside/outside" status for a set of vertices VERT and a general
+    polygon (PSLG) embedded in the two-dimensional plane. General non-convex and
+    multiply-connected polygonal regions can be handled.
+  doc_url: https://github.com/dengwirda/inpoly-python/blob/master/README.md
+  dev_url: https://github.com/dengwirda/inpoly-python
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - dengwirda
```
Comment (xylar) [inline recipes/inpoly/meta.yaml:67]: @dengwirda, could you acknowledge that you are willing to be a maintainer of this package?

**Comment by xylar** [comment]
The license is custom, similar to that for `jigsawpy`: https://github.com/conda-forge/jigsawpy-feedstock/blob/master/recipe/meta.yaml#L52

File: recipes/inpoly/meta.yaml (hunk 1 above)
Comment (dengwirda) [reply recipes/inpoly/meta.yaml:67]: Yes, very happy to do so @xylar.

**Comment by xylar** [comment]
@conda-forge/help-python, this is ready to review.  Please let us know if there are any concerns.

**Comment by xylar** [comment]
@conda-forge/help-python, I'm sure you're quite busy.  Just wanted to make sure this PR doesn't get lost.

**Comment by xylar** [comment]
Thanks @beckermr!

**Pull request merged**